Add Sybil labels to wallet labels list (#113)
getformo/docs.formo.so: features/wallet-intelligence/wallet-labels.mdx

| **Coinbase One** | Coinbase One subscription attestations |
| **Binance Verified** | Holds a [Binance Account Bound Token (BABT)](https://www.binance.com/en/babt), attesting the wallet completed Binance KYC |

### Sybil

Wallets that appear on published airdrop sybil / farmer lists. These are one-time historical snapshots from past airdrop investigations, so a wallet is labelled only if it was named in the original list.

| Label | Description |
|-------|-------------|
| **Sybil (LayerZero)** | Named in the [LayerZero airdrop sybil list](https://github.com/psrvere/layerzero-sybil-checker) as a suspected airdrop farmer |

> | **Sybil (Hop)** | Listed as an eliminated sybil attacker in the [Hop Protocol airdrop](https://github.com/hop-protocol/hop-airdrop/blob/master/src/data/eliminatedSybilAttackers.csv) |

Reviewer: ```suggestion
| **Sybil (Hop)** | Listed as a sybil attacker in the [Hop Protocol airdrop](https://github.com/hop-protocol/hop-airdrop/blob/master/src/data/eliminatedSybilAttackers.csv) |
```
